src/drive/web/modules/filelist/headers/CarbonCopy.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import { TableHeader } from 'cozy-ui/transpiled/react/Table'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import CarbonCopyIcon from 'cozy-ui/transpiled/react/Icons/CarbonCopy'
+
+import CertificationTooltip from 'drive/web/modules/certifications/CertificationTooltip'
+
+import styles from 'drive/styles/filelist.styl'
+
+const CarbonCopyHeader = () => {
+  const { t } = useI18n()
+
+  return (
+    <TableHeader className={styles['fil-content-header']}>
```

Comment:
Nit: If we could have a ContentTableHeader exported somewhere, we could dispense with the filelist.styl import.

Reply:
Oui on pourrait aussi le faire pour les cells qui ont toutes un `<TableCell className={cx(styles['fil-content-cell']` en commun. Il y a un chantier de refacto sur ce tableau à mener, qui a été entamé par l'extraction de ces headers et cell depuis File dans des composants à part. Je pense qu'on va continuer le travail dans des prochaines PR.